API-reference.md

```diff
@@ -129,6 +129,26 @@ Immediately send all queued telemetry. Synchronous.
 
 You don't usually have to use this, as it happens automatically on window closing.
 
+<a name="setAuthenticatedUserContext"></a>
+### setAuthenticatedUserContext
+
+    setAuthenticatedUserContext(authenticatedUserId: string, accountId?: string)
+
+Set the authenticated user id and the account id in this session. Use this when you have identified a specific signed-in user. Parameters must not contain spaces or ,;=|
```

Comment:
It's worth mentioning that it'll be saved as a cookie and sent to server, where the server side SDK - if installed - will take it and attach to server side telemetry.